akka-actor-tests/src/test/java/akka/actor/ActorSystemTest.java

```diff
@@ -1,4 +1,8 @@
- /**
+/*
+ * Copyright (C) 2018 Lightbend Inc. <https://www.lightbend.com>
+ */
+
+/**
   * Copyright (C) 2009-2018 Lightbend Inc. <https://www.lightbend.com>
   */ 
```

Comment:
looks wrong?

Reply:
Probably something due to extra space in the beginning of each line. I would actually expect MiMa to format it correctly or complain about it. Anyway this could be also fixed with a bit of enhancing of detection regexp.